packages/legacy-events/ResponderEventPlugin.js

```diff
@@ -158,6 +151,91 @@ const eventTypes = {
 // Start of inline: the below functions were inlined from
 // EventPropagator.js, as they deviated from ReactDOM's newer
 // implementations.
+
+function getParent(inst) {
```

Comment:
I get that this is 'legacy', but it's weird that fibers are called instances here (since we use that word for something else elsewhere. 

could you explain *why* this other traversal fns are inlined multiple times instead of in one module? Also why do some of them have types, and some don't?

Reply:
They're inlined because we don't want them in ReactDOM anymore and by pulling them out, it means we won't accidently break another module by changing something else. They're essentially legacy, and in a bunch of the cases, will likely be killed when their associated module drops them. Some have types because the module is Flow types, whilst before they came from a module that wasn't Flow typed.